posthog/temporal/data_imports/sources/README.md

```diff
@@ -3,7 +3,7 @@
 Adding a new source should be pretty simple. We've refactored the sources so that you need to only add your source logic and update a minimal amount of other files. Below is a step-by-step guide:
 
 1. Add a new enum value to `ExternalDataSourceType` (posthog/warehouse/types.py). The key should be fully capitalized and the value should be in pascal case.
-2. Run django migrations - `DEBUG=1 ./bin/migrate`
+2. Run django migrations - `DEBUG=1 python manage.py makemigrations && ./bin/migrate`
```

Comment:
@Gilbert09 this was missing here on the steps, everything worked great besides this!